app/src/main/java/com/ucmobiledevelopment/freeways/MainViewModel.kt

```diff
@@ -60,7 +60,7 @@ class MainViewModel(var incidentService : IIncidentService = IncidentService())
 
     fun fetchIncidents(fromCaseYear: Int, toCaseYear: Int, state: Int, county: Int){
         viewModelScope.launch {
-           var innerIncidents = incidentService.fetchIncidents(fromCaseYear, toCaseYear, state, county)
+           val innerIncidents = incidentService.fetchIncidents(fromCaseYear, toCaseYear, state, county)
```

Comment:
A good suggestion, though it appeared in a previous review. When that happens, first in wins.